staging/operator-lifecycle-manager/pkg/controller/operators/catalog/subscription/syncer.go

```diff
@@ -89,18 +81,28 @@ func (s *subscriptionSyncer) Sync(ctx context.Context, event kubestate.ResourceE
 	return nil
 }
 
-func (s *subscriptionSyncer) Notify(event kubestate.ResourceEvent) {
+func (s *subscriptionSyncer) Notify(event types.NamespacedName) {
 	s.notify(event)
 }
 
 // catalogSubscriptionKeys returns the set of explicit subscription keys, cluster-wide, that are possibly affected by catalogs in the given namespace.
-func (s *subscriptionSyncer) catalogSubscriptionKeys(namespace string) ([]string, error) {
-	var keys []string
+func (s *subscriptionSyncer) catalogSubscriptionKeys(namespace string) ([]types.NamespacedName, error) {
+	var cacheKeys []string
 	var err error
 	if namespace == s.globalCatalogNamespace {
-		keys = s.subscriptionCache.ListKeys()
+		cacheKeys = s.subscriptionCache.ListKeys()
 	} else {
-		keys, err = s.subscriptionCache.IndexKeys(cache.NamespaceIndex, namespace)
+		cacheKeys, err = s.subscriptionCache.IndexKeys(cache.NamespaceIndex, namespace)
+	}
+
+	keys := make([]types.NamespacedName, len(cacheKeys))
```

Comment:
I messed this one up:
```suggestion
	keys := make([]types.NamespacedName, 0, len(cacheKeys))
```